### DISCUSSION TITLE
A question about swarm2starcamera

### DISCUSSION THREAD

**Discussion by cordaX** [groops-devs/groops#301] [Q&A]
### 
Hello,

I find something wrong using swarm2starcamera. I use SWARM star camera cdf data(STRAATT.*MDR_SAT_AT.cdf) as inputfile. It shows
convert error '\string.cpp:123 (toInt): cannot read integer from string? '  
It seems can't match the timestamp properly ( as shown in fig below).
![71f3ff465a1c9274e52480fe64f88f0](https://github.com/user-attachments/assets/84192f47-32f3-4c0f-8504-b489befcee8c)
How do I use the program properly？
Thank you in advance for your feedback and help.
With Kind Regards
Corda

**Comment by tmayerguerr** [comment]
Hi Corda,

GROOPS cannot read CDF format directly, but the file must be first converted with the program cdfexport provided by the Goddard Space Flight Center (http://cdf.gsfc.nasa.gov/). 

We run the conversion program with something like:
```cdfexport -settings 020cdfExport.set -batch text -text swarm1_starCamera_xxxx-xx-xx.txt  STRAATT.*MDR_SAT_AT.cdf;```
and the settings file [020cdfExport.set.zip](https://github.com/user-attachments/files/18259512/020cdfExport.set.zip).

regards
Torsten
